consensus/maintenance.go

```diff
@@ -96,14 +95,13 @@ func (s *State) applyContractMaintenance(bn *blockNode) {
 	// Expiring a contract deletes it from the map we are iterating through, so
 	// we need to store it and deleted once we're done iterating through the
 	// map.
+	currentHeight := s.height()
 	var expiredContracts []FileContractID
 	for id, contract := range s.fileContracts {
-		if s.height() == contract.Expiration {
+		if contract.Expiration == currentHeight {
```

Comment:
please rename from `contract` to `fc`.

I know it was like this when you found it just asking for a rename.